packages/xpetra/src/Vector/Xpetra_TpetraVector_def.hpp

```diff
@@ -174,7 +174,10 @@ template<class Scalar, class LocalOrdinal, class GlobalOrdinal, class Node>
     typedef EpetraNode Node;
 
 #undef XPETRA_TPETRAMULTIVECTOR_SHORT
```

Comment:
Since we are pulling in the `Xpetra_TpetraMultiVector.hpp`, why do we need to do the "short names dance"?

Reply:
It may be used in the `_def`  I didn't check whether it is needed or not.  I can see the value in short names, but they also make it easy to shoot yourself in the foot.